components/ide/xterm/leeway.Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+FROM node:16 as ide_installer
+
+ARG XTERM_COMMIT
+
+RUN apt update -y \
+    && apt install python3 --no-install-recommends -y
+
+RUN mkdir /build \
```

Comment:
@filiptronicek looks good

Reply:
Tried to build one, it works well https://github.com/gitpod-io/gitpod/pull/19652#issuecomment-2079917591